src/pyseekdb/client/client_seekdb_server.py

```
@@ -78,190 +79,204 @@ def _ensure_connection(self) -> pymysql.Connection:
         Note:
             Remote server has multi-tenant architecture. Database is scoped to client's tenant.
         """
         if tenant != self.tenant and tenant != DEFAULT_TENANT:
-            logger.warning(f"Specified tenant '{tenant}' differs from client tenant '{self.tenant}', using client tenant")
-        
+            logger.warning(
+                f"Specified tenant '{tenant}' differs from client tenant '{self.tenant}', using client tenant"
+            )
+
         logger.info(f"Getting database: {name} in tenant: {self.tenant}")
         sql = f"SELECT SCHEMA_NAME, DEFAULT_CHARACTER_SET_NAME, DEFAULT_COLLATION_NAME FROM information_schema.SCHEMATA WHERE SCHEMA_NAME = '{name}'"
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)